dashboards/resources.libsonnet

```diff
@@ -2,53 +2,153 @@ local g = import 'grafana-builder/grafana.libsonnet';
 
 {
   grafanaDashboards+:: {
+    'k8s-resources-multicluster.json':
+      local tableStyles = {
+        cluster: { //TODO: Does this need to be "%(clusterLabel)s" ??
```

Comment:
Why would this possibly need to be "%(clusterLabel)s"?

Reply:
the literal "cluster" seen at left here has to match the "group by (...)" clause from the promQL queries. Quickly tested this and confirmed that left side should be the clusterLabel value from config.